packages/preset-mini/src/_rules/border.ts

```diff
@@ -80,6 +80,23 @@ function borderColorResolver(direction: string) {
       }
     }
     else if (color) {
+      if (/\[[^\]]*\]/.test(name)) {
+        const multipleValue = name.slice(1, -1).split('_')
+        if (multipleValue.length) {
+          return multipleValue.reduce((result, v) => {
```

Comment:
I'd rather let syntax with spaces not get processed further.

Reply:
What are you referring to? Can you give me a specific example?

Reply:
like when using `b-[a_b_c]` it should just resolve to `border: a b c` since `a`, `b`, and `c` can appear in any order. See
https://developer.mozilla.org/en-US/docs/Web/CSS/border#formal_syntax
https://developer.mozilla.org/en-US/docs/Web/CSS/Value_definition_syntax#double_bar

Reply:
Do you want to `say b-[a_b_c]`, is there something wrong with the order of generation, or do you think the corresponding `border-width`、`border-style`、`border-color` should not be generated but merged into attributes in the `border`?